File: frontend/src/app/dto/exam-collection/exam-collection.insert.res.dto.ts
import { ExamCollectionInsertDataResDto } from "./exam-collection.insert.data.res.dto";

export class ExamCollectionInsertResDto{
    private _data! : ExamCollectionInsertDataResDto
	private _message! : string

    public get data(): ExamCollectionInsertDataResDto {
        return this._data;
    }

    public set data(data: ExamCollectionInsertDataResDto) {
        this._data = data;
    }

    public get message(): string {
        return this._message;
    }

    public set message(message: string) {
        this._message = message;
    }

}

File: backend/src/main/java/com/lawencon/elearn/dto/examcollection/ExamCollectionInsertResDto.java
package com.lawencon.elearn.dto.examcollection;

public class ExamCollectionInsertResDto {
	private ExamCollectionInsertDataResDto data;
	private String message;
	public ExamCollectionInsertDataResDto getData() {
		return data;
	}
	public void setData(ExamCollectionInsertDataResDto data) {
		this.data = data;
	}
	public String getMessage() {
		return message;
	}
	public void setMessage(String message) {
		this.message = message;
	}
}
